### PR TITLE
Add checkbox to create empty project

### PR DESCRIPTION
**Pull request by HunseopJeong**
Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: public/js/project.js [hunk 1]
```diff
@@ -181,24 +193,37 @@ $('#new-project').on('show.bs.modal', function() {
...
+          // Skip profile-version on WASM Module
+          updateTemplateSample(projectFormat, projectType);
+          templateSampleDlg.show();
+          currentProcess = 'template-sample';
+        }
       }
 
       break;
     case 'profile-version':
+        profileVersionDlg.hide();
```
Comment (ws29jung) [inline public/js/project.js]: aligne? shouldn't be 2 space?

File: public/js/project.js [hunk 2]
```diff
@@ -86,6 +87,16 @@ $('#new-project').on('show.bs.modal', function() {
...
   });
 
+  $("#empty-checkbox").on('change', function() {
+    emptyProject = this.checked;
+    if (emptyProject) {
+      templateBtn.removeClass('btn-primary');
+      sampleBtn.removeClass('btn-primary');
+    } else {
+      templateBtn.addClass('btn-primary');
+    }
```
Comment (ws29jung) [inline public/js/project.js]: What about sampleBtn?

File: public/js/project.js (hunk 1 above)
Comment (HunseopJeong) [reply public/js/project.js]: Fixed. Thanks.

File: public/js/project.js (hunk 2 above)
Comment (HunseopJeong) [reply public/js/project.js]: sampleBtn? We deselect the templateBtn and the sampleBtn when clicking the empty checkbox.
On the contrary, If we deselect the empty checkbox then activating templateBtn.

**Comment by Kwanghyuk-Kim** [comment]
What about adding the reason of this patch to commit message ?
Why empty project creation is necessary ?

**Comment by HunseopJeong** [comment]
If I want to create my application, I create project based on the supported template or sample and then remove all files :(
So I want to support empty project.

File: public/js/project.js (hunk 2 above)
Comment (hyundukkim) [reply public/js/project.js]: In case of 'emptyProject == false', should only templateBtn be activated ?

Comment (HunseopJeong) [reply public/js/project.js]: Yes. That is default.

Comment (hyundukkim) [reply public/js/project.js]: OK.

**Pull request merged**